### DISCUSSION TITLE
Welcome to PyNARS Discussions!

### DISCUSSION THREAD

**Discussion by bowen-xu** [opennars/OpenNARS-4#22] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kripper** [comment]
Congratulations @bowen-xu, I have been checking all available NARS resources and implementations, and believe that PyNARS is the future. You are doing a great job.

I hope at some point the team cleans all the NARS information mess spread around the net and centralizes all the efforts here on GitHub so new developers and researchers are not scared away by the information chaos and outdated resources, and have a better experience joining the community.

**Reply by bowen-xu** [reply]
Thanks! PyNARS is a team work, and we have several members working on it.
There are indeed many implementations of NARS; I agree that it may scare away some beginners. We hope to make a 'standard' implementation as a start point for beginners, as well as a reference for other researchers who want to implement their own version. We'll consider your advice on cleaning the information.
